e2e/cypress/integration/messaging/emoji_keyboard_entry_spec.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// [number] indicates a test step (e.g. 1. Go to a page)
+// [*] indicates an assertion (e.g. * Check the title)
+// Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Message', () => {
+    beforeEach(() => {
+        // # Login as "user-1" and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M16738 Use keyboard navigation in emoji picker', () => {
+        // # Open emoji picker
+        cy.get('span[aria-label="Open emoji picker"]').click();
+
+        // # Wait for emojis to load
+        cy.get('#emojiPicker').should('be.visible');
+
+        // # Randomize directional arrow inputs
+        const min = 1;
+        const max = 5;
+        for (const direction of ['right', 'down', 'up']) {
+            const pressCount = Math.floor(Math.random() * max) + min;
+
+            for (let i = 0; i < pressCount; i += 1) {
+                cy.get('input[aria-label="Search for an emoji"]').type(`{${direction}arrow}`);
+            }
+        }
+
+        cy.get('.emoji-picker__preview-image-label-box .emoji-picker__preview-aliases').invoke('text').then((selectedEmoji) => {
+            // # Select chosen emoji
+            cy.get('input[aria-label="Search for an emoji"]').type('{enter}');
+
+            // # Post message with keyboard
+            cy.get('#post_textbox').type('{enter}');
+
+            // # Compare selected emoji with last post
+            cy.getLastPost().find('.post-message__text-container .all-emoji .emoticon').should('have.attr', 'title', selectedEmoji);
```

Comment:
You can probably simplify this selector to just `cy.getLastPost().find('.emoticon').should(blahblah)` as `.emoticon` should only have one instance under the last post.

Reply:
You're right, the selection chain was from when I was debugging prior to discovering getLastPost().  Updated in a forthcoming commit.

Reply:
Fixed.  The selection is a bit loose for my liking but it is alright given the root of the selection is  rooted at the last post.